styles/style.css

```diff
@@ -90,10 +119,33 @@ body {
 }
 
 /* Article */
+
+.my-box {
+	padding: 3rem 0;
+	
+}	
+
+.my-box .box {
+	border: 1px solid #292b2c;
+	min-height: 7rem;
+	font-size: 27px;
+	font-weight: 700;
+	text-align: center;
+	margin: 1rem;
```

Comment:
You cannot add margin on a level that also has a `col-` class, it will interfere with the grid.